src/tests/system/tests/test_memory_cache.py

```diff
@@ -1320,7 +1322,7 @@ def test_memory_cache__invalidate_groups_before_stop(client: Client, provider: G
 @pytest.mark.importance("high")
 @pytest.mark.cache
 @pytest.mark.topology(KnownTopologyGroup.AnyProvider)
-def test_memory_cache__invalidate_groups_after_stop(client: Client, provider: GenericProvider):
+def test_memory_cache__invalidate_groups_cache_after_stop(client: Client, provider: GenericProvider):
```

Comment:
"invalidate_group_cache"

Reply:
The previous test is group_cache and then this one is groups_cache since it tests multiple groups. These tests need to be rework.